src/doc/yaml/enrollment.yaml

```diff
@@ -0,0 +1,652 @@
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      responses:
+        '204':
+          description: Update successful
+        '400':
+          description: Bad request
+        '403':
+          description: Update not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+                
+
+  /v1/enrollments:
+    post:
+      tags:
+        - Enrollment
+      summary: Retrieve a list of enrollments which match passed in search criteria
+      operationId: findEnrollments
+      parameters:
+        - name: expressions
+          in: query
+          description: The expressions to evaluate.
+          required: true
+          schema:
+            $ref: '#/components/schemas/Expressions'
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: A set of expressions on attributes of the person
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Expressions'
+            example:
+              - attributeName: firstName
+                operator: "="
+                value: John
+              - attributeName: dateOfBirth
+                operator: "<"
+                value: "1990-12-31"
+      responses:
+        '200':
+          description: Read successful
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                    $ref: '#/components/schemas/Enrollment'
+        '400':
+          description: Bad request
+        '403':
+          description: Read not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+                
+  /v1/enrollment/{enrollmentId}/buffer:
+    post:
+      tags:
+        - Enrollment
+      summary: This service is used to send separately the buffers of the images
+      operationId: sendBuffer
+      parameters:
+        - name: enrollmentId
+          in: path
+          description: the id of the enrollment
+          required: true
+          schema:
+            type: string
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: The image of the request
+        required: true
+        content:
+          plain/text:
+            schema:
+              type: string
+              format: byte
```

Comment:
We should favour binary transfer using type application/octet-stream and/or image/jpeg and/or ... Format should be "binary".
See https://swagger.io/docs/specification/describing-request-body/file-upload/ for more info.

Reply:
I will go for  application/octet-stream